Base/usr/share/man/man1/CharacterMap.md

```diff
@@ -1,6 +1,6 @@
 ## Name
 
-![Icon](/res/icons/16x16/app-character-map.png) Character Map
+![Icon](file:///res/icons/16x16/app-character-map.png) Character Map
```

Comment:
This breaks `man.serenityos.org`. See: https://github.com/SerenityOS/serenity/pull/11965

Reply:
Then we need to remove these from markdown-check (pre-commit is broken after #11965), right now i just chmod -x the pre-commit hook

Reply:
How about we fix the markdown check to ignore    `/res` image links for man pages?